tensorboard/plugins/profile_redirect/tf_profile_redirect_dashboard/tf-profile-redirect-dashboard.html

```diff
@@ -0,0 +1,110 @@
+<!--
+@license
+Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../paper-button/paper-button.html" />
+<link rel="import" href="../tf-imports/polymer.html" />
+
+<!--
+  A frontend that directs users to upgrade to the new version of the
+  profile plugin, now distributed separately from TensorBoard.
+-->
+<dom-module id="tf-profile-redirect-dashboard">
+  <template>
+    <div class="message">
+      <h3>The profile plugin has moved.</h3>
+      <p>
+        Please install the new version of the profile plugin from PyPI by
+        running the following command from the machine running TensorBoard:
+      </p>
+      <textarea id="commandTextarea" readonly on-blur="_removeCopiedMessage">
+[[_installCommand]]</textarea
+      >
+      <div id="copyContainer">
+        <span id="copiedMessage"></span>
+        <paper-button raised on-tap="_copyInstallCommand"
+          >Copy to clipboard</paper-button
+        >
+      </div>
+    </div>
+
+    <style include="dashboard-style"></style>
```

Comment:
Omit this unless you are making use of it. Else, I believe you need to import https://github.com/tensorflow/tensorboard/blob/master/tensorboard/components/tf_dashboard_common/dashboard-style.html in rel="import".